funcx_endpoint/tests/unit/test_endpoint_unit.py

```diff
@@ -0,0 +1,66 @@
+import functools
+import io
+import random
+import uuid
+from collections import namedtuple
+from unittest import mock
+
+import pytest
+
+from funcx_endpoint.endpoint.endpoint import Endpoint
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Is this necessary? Looks like every test in this file uses this fixture anyway

Reply:
Ah, good catch.  From an early thought-process while testing.